### DISCUSSION TITLE
Odd behavior after switching to latest Ender3V2  firmware

### DISCUSSION THREAD

**Discussion by scsilvers** [mriscoc/Ender3V2S1#326] [Q&A]
I've got 3 Ender3V2's (with the 4.2.2 board and OctoPrint 1.8.1 servers) that I maintain for a local makerspace.  I've had them running Jyers for the last few months and things have been ok, but I wanted to switch over to mriscoc's firmware after reading about some of the extra capabilities it provides.   I switched the best of my machines over a couple of days ago and while most things are great, I'm having an issue with a bulge forming in the center of the objects I'm printing.   The first one of these I printed was a touch small on the Z axis (19.6 mm vs 20.2 on the x and y), so I adjusted the esteps and the subsequent two that I printed are almost spot on 20mm.  The bulge is still the same through all 3.  I ran the same gcode file I used for these on one of my other machines and it's a clean print, so I don't think it's something about the file itself, unless there's something specifically on those center lines of gcode that's causing the problem.
Has anyone else run into this issue or have some ideas on how to troubleshoot?

![PXL_20220826_211059298](https://user-images.githubusercontent.com/109870822/187042546-00adf35a-49d5-4693-bb39-813d3343c8bf.jpg)

**Comment by mriscoc** [comment]
This firmware unlocks higher speeds and accelerations by default, normal firmware limits the print speed despite the values of the axis feed rates in the G-code file. Try lowering those limits in the start g-code script, use the values provided in the wiki for the start g-code and end-gcode as a reference.

**Reply by scsilvers** [reply]
I thought I had done that already, but some of the lines were missing from my slicer so I added them back in.   It improved the print.  I guess I'll need to play with those settings some more.
Thanks 
![PXL_20220827_205523911](https://user-images.githubusercontent.com/109870822/187047742-678308e7-efa7-427c-b0ce-413e02681b29.jpg)
